examples/IB/explicit/ex7/example.cpp

```diff
@@ -54,6 +54,28 @@
  *                                                                             *
  *******************************************************************************/
 
+struct Lag_Coords
+{
+    int lag_idx; // The global lag index
+    double X;
+    double Y;
+    double Z;
+
+    // Default constructor
+    Lag_Coords(){};
```

Comment:
```suggestion
    Lag_Coords()
    : lag_idx(std::numeric_limits<int>::max()
    , X(std::numeric_limits<double>::quiet_NaN()
    , Y(std::numeric_limits<double>::quiet_NaN()
    , Z(std::numeric_limits<double>::quiet_NaN()
    {}
```
we should put in invalid values to prevent accidentally reading an uninitialized structure.